introduction.md

```diff
@@ -9,4 +9,5 @@ My deadly skills
 *JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?